src/smepmp.adoc

```diff
@@ -1,5 +1,5 @@
 [[smepmp]]
-== PMP Enhancements for memory access and execution prevention on Machine mode (Smepmp)
+== "Smepmp" Enhancements for memory access and execution prevention on Machine mode, Version 1.0.0
```

Comment:
"Enhancements" should be changed to "Extension" (and it should be singular).

Reply:
Note that Smepmp makes enhancements on top of the PMP spec.  So maybe the title should be:

"Smepmp" Extension for PMP enhancements for memory access and execution prevention in Machine mode, Version 1.0.0

Reply:
Works for me.

Reply:
Fixed.  Thanks.